### PR TITLE
back end server

### PR DESCRIPTION
**Pull request by kavitharaju**
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-box3/graphql-catalog-next/12)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (mannycolon) [inline package.json]: Should we remove this file?

File: server/schema/resolver.js [hunk 2]
```diff
@@ -1,19 +1,38 @@
 const { GraphQLScalarType } = require('graphql');
 
+dummyOrgs = [
```
Comment (mannycolon) [inline server/schema/resolver.js]: Should we remove the dummyOrgs?

File: package.json (hunk 1 above)
Comment (kavitharaju) [reply package.json]: As, both client and server has seperate installations, I dont think this is required.

File: server/schema/resolver.js (hunk 2 above)
Comment (kavitharaju) [reply server/schema/resolver.js]: it is not used. Yes,  I shall remove it

**Review by mannycolon — APPROVED** [review]
Great job!

**Pull request merged**